lib/Migrate_ast.mli

```diff
@@ -98,6 +98,8 @@ module Traverse : sig
     | Structure : Parsetree.structure fragment
     | Signature : Parsetree.signature fragment
     | Use_file : Parsetree.toplevel_phrase list fragment
+    | Core_type : Parsetree.core_type fragment
+    | Module_type : Parsetree.module_type fragment
```

Comment:
Is this enough ?
For example, formatting expressions would be handy for Mdx: https://github.com/realworldocaml/mdx/pull/316/files#diff-7436767382485ab85681e67eda27b5af703dddc879b9605ec4390bd7042fb47cR113

Reply:
Done! it should simplify the mdx integration indeed